software/software.mk

```diff
@@ -1,7 +1,10 @@
 defmacro:=-D
 incdir:=-I
+include $(ROOT_DIR)/core.mk
 include $(ROOT_DIR)/system.mk
 
+PREFIX:=$(CORE_NAME)_
```

Comment:
useless renaming, use CORE_NAME directly

Reply:
It is not useless. If I didn't use it like this, it just doesn't work because of the inclusion of embedded.mk files in firmware Makefile will overwrite the variable CORE_NAME. So I found this simple workaround.